module/behaviour/planning/KickPlanner/src/KickPlanner.hpp

```diff
@@ -36,6 +36,7 @@ namespace behaviour {
 
         private:
             bool kickValid(const arma::vec3& ballPos);
```

Comment:
what is this even doing? is it saying that if the arma::vec is null then it's false?? 
could be out of the scope of this PR, but this line makes no sense to me